src/components/Mailbox.vue

```diff
@@ -61,6 +62,10 @@ export default {
 	},
 	mixins: [isMobile],
 	props: {
+		loadMoreLabel: {
+			type: String,
+			default: '',
```

Comment:
nit: default undefined so the EnvelopeList default is used when nothing is passed?